source/shared/net/desktop-wallet/add-baker-pool-dw.rst

```diff
@@ -0,0 +1,146 @@
+.. _add-baker-pool-dw:
+
+===================================
+Open a baker pool in Desktop Wallet
+===================================
+
+.. Note::
+
+   Currently delegation is only available on Testnet.
+
+You can open a baker pool so that others can delegate stake from an account to your baker pool. When others delegate some stake to a baker pool, it can influence the chances of the baker owner of the baker pool being selected to bake a block and receive baker rewards. See :ref:`Baker management<baker-pool>` for information about how to be a responsible baker.
+
+.. Note::
+
+   All transfers and transactions cost a fee, including staking and unstaking transactions. The fee is based on the set NRG for that transaction and the current exchange rate.
+   The cost of transaction fees is stable in Euros, and therefore the price in CCD varies depending on the CCD to EUR exchange rate. The fee will always be deducted from the **Balance** of the account, so it is important to have some available CCDs to cover fees. A locked-for-staking balance cannot be used to pay for these transactions.
+   You can see the fee in the transaction log.
+
+.. Warning::
+   Make sure you have enough funds in your disposable balance to cover transaction fees.
+
+Open a baker pool (Single-signature account)
```

Comment:
You don't necessarily use this to only open a baker pool, it can also be used to close it or just change the url (or commission if we could).
Maybe this should just be "update baker pool" maybe :shrug:

Reply:
I think it is best to leave it. Even though the action is configuring their baker most users won't grasp that concept. They see it as doing something for the first time. The button name is correct in the steps.